models/users.model.js

```diff
@@ -0,0 +1,16 @@
+const fs = require("fs/promises");
+const db = require("../db/connection")
+
+
+const fetchUsers = () => {
+    const query = `SELECT * FROM users;`
+    return db.query(query).then((result) => {
+        if (result.rows.length === 0){
+          return Promise.reject({ status: 404, msg: 'No Users found'})
+        }
```

Comment:
🚨 This is untested code!
Also, is it necessary? If the `response.body.users` was an empty array, that tells me no users in a way! Also will make it easier for your front end project if you don't throw an error here (_spoilers_) 😄